DDDEastAnglia/DataAccess/DefaultSponsorSorter.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DDDEastAnglia.DataAccess.SimpleData.Models;
+
+namespace DDDEastAnglia.DataAccess
+{
+    public static class SponsorExtensions
+    {
+        public static IEnumerable<Sponsor> OrderBySponsorSorter(this IEnumerable<Sponsor> sponsors)
```

Comment:
See previous comment about pure function.. but if we go with a class then I like the idea of

`SortUsing<DefaultSponsorSorter>()`

or if we do inject it.. then

`SortUsing(injectedSorter)`

Any preference?